docs/source/upcoming_release_notes/1240-mfx_6crystal_spec.rst

```diff
@@ -0,0 +1,31 @@
+1240 mfx_6crystal_spec
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- VonHamos6Crystal
+- VonHamosCrystal_2
```

Comment:
I think along with disambiguating the two crystals in the code docstrings, we could use a brief explanation here